scripts/tokenomics-emulation/network.ts

```diff
@@ -46,23 +46,15 @@ class Network {
     sender: KeyringPair,
     destination: string,
     value: string
-  ): Promise<string> {
+  ): Promise<any> {
     const amount = +value * 10 ** this.config.network.decimals;
     console.log(
       `About to transfer ${amount} native assets to ${destination} from ${sender.address}\n`
     );
     const { nonce } = await this.api.query.system.account(sender.address);
 
     const transfer = this.api.tx.balances.transfer(destination, amount);
-    return new Promise((res, rej) => {
-      transfer
-        .signAndSend(
-          sender,
-          { nonce: nonce },
-          Network.sendStatusCb.bind(this, res, rej)
-        )
-        .catch((err) => rej(err));
-    });
+    return transfer;
```

Comment:
Let's leave the ability to execute this transaction without batch.

Just move the transaction creation part to a separate function and reuse it here and in batch preparation.

Reply:
Added a new function to sign and send single transaction